selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/intelligencecommand/CheckIntelligenceCommandFromToolbarTest.java

```diff
@@ -62,6 +63,12 @@ public void launchClonedWepAppTest() throws Exception {
     projectExplorer.waitItem(PROJECT_NAME);
     commandsToolbar.clickWithHoldAndLaunchCommandFromList(PROJECT_NAME + ": build and run");
     consoles.waitExpectedTextIntoConsole(" Server startup in");
+
+    String previewUrl = consoles.getPreviewUrl();
+    if (previewUrl.contains("route")) {
+      WaitUtils.sleepQuietly(10);
```

Comment:
Could you, please, add comment here with explanation what such a big timeout is for?

Reply:
It need when test is running on che6-ocp platform

Reply:
I see. Please, add comment into the code to make it clear for other developers.

Reply:
Have done